electrum/submarine_swaps.py

```diff
@@ -178,6 +176,30 @@ def start_network(self, *, network: 'Network', lnwatcher: 'LNWalletWatcher'):
                 continue
             self.add_lnwatcher_callback(swap)
 
+    async def pay_pending_invoices(self):
+        # for server
+        self.invoices_to_pay = set()
+        while True:
+            await asyncio.sleep(1)
+            for key in list(self.invoices_to_pay):
+                swap = self.swaps.get(key)
+                if not swap:
+                    continue
+                invoice = self.wallet.get_invoice(key)
+                if not invoice:
+                    continue
+                current_height = self.network.get_local_height()
+                delta = current_height - swap.locktime
+                if delta > - 5:
+                    # fixme: should consider cltv of ln payment
+                    self.logger.info('locktime too close {key}')
+                    continue
+                success, log = await self.lnworker.pay_invoice(invoice.lightning_invoice, attempts=1)
+                if not success:
+                    self.logger.info('failed to pay invoice {key}')
```

Comment:
```suggestion
                    self.logger.info(f'failed to pay invoice {key}')
```
same, ~4 lines above